tests/sles4sap/publiccloud/qesap_prevalidate.pm

```diff
@@ -63,7 +64,14 @@ sub run {
         $self->{my_instance} = $instance;
         last if ($instance->{instance_id} =~ m/vmhana/);
     }
-    $self->wait_for_cluster(wait_time => 60, max_retries => 10);
+
+    # Softfail 12sp5 due to https://bugzilla.suse.com/show_bug.cgi?id=1233026
+    if (is_sle('12+')) {
+        $self->wait_for_cluster(wait_time => 60, max_retries => 10, proceed_on_failure => 1);
+    }
+    else {
+        $self->wait_for_cluster(wait_time => 60, max_retries => 10);
+    }
```

Comment:
A little Perl-rific (or is it more like C Language?) suggestion:
```
$self->wait_for_cluster(wait_time => 60, max_retries => 10, proceed_on_failure => is_sle('=12-SP5'));
```
Instead of the if-else block :)

Reply:
That's a great suggestion! In the end I deciided to do all the handling in prevalidate, so if-else is unavoidable, but a great suggestion nonetheless